### DISCUSSION TITLE
Correct way of fetching custom model via frontend api?

### DISCUSSION THREAD

**Discussion by bakerhaider** [swellstores/community#475] [Help] [answered]
![ae2b49f43bb6bc34511752efee7d7bf9](https://github.com/swellstores/community/assets/33936834/da0acc5e-83f4-4b28-bb03-52fa4c5614a4)
![bd9423de42cfe6aef7c4f2da1217134f](https://github.com/swellstores/community/assets/33936834/6c542006-5891-4007-a798-9d3e4e0e1245)

**Comment by sags95** [comment, marked as answer]
@BakerAlblam If you're models are created with the `content` namespace then you can fetch them using swell-js with `swell.content.get('custom-product-file)` or `swell.content.get('custom-product-file', <id>)`

**Reply by bakerhaider** [reply]
hi, thanks for the reply, id appreciate if you could tell me if its possible to make the customers upload files for along with their order as we need the files for the logos we produce. Thanks

**Reply by bakerhaider** [reply]
the important part here is the ability to upload assets by the customer

**Reply by sags95** [reply]
It's possible but file upload would have to be done through the Backend API, not sure if you've created a custom field but here is a response that details how to upload files

https://github.com/orgs/swellstores/discussions/3#discussioncomment-4242550

**Reply by bakerhaider** [reply]
Thanks alot!
